.gitignore

```diff
@@ -5,10 +5,15 @@ tags/
 *.pyc
 *.bak
 BuildConfig.conf
+/Conf
+/.vscode
+SharedCrypto_extdep/
 
 # Ignore SelfDescribingEnvironment artifacts.
 *_extdep/
 
 # Ignore cloned dependencies
 /Common
 /MU_BASECORE
+.NugetOutput/
```

Comment:
these also look like local dev relics

Reply:
When publishing a driver, we aggregate the files needed into a new folder called .NugetOutput, maybe this folder should be inside the build folder

Reply:
Okay it now gets put in the build directory